kieker-analysis/build.gradle

```diff
@@ -22,6 +23,11 @@ dependencies {
 	compile 'com.google.guava:guava:20.0'				// (21.0 and above require java 8) used by StringRegistryCache for amqp reader
 	compile 'org.tukaani:xz:1.6'	// decompression algorithms used by the readers
 		
+	compile 'org.nanohttpd:nanohttpd:2.3.0'
+			
+	compile 'org.slf4j:slf4j-api:1.7.25'
```

Comment:
As discussed, do we really want to include a logging implementation in Kieker Analysis?

Reply:
Well org.slf4j:slf4j-api:1.7.25 is only API, but I guess you mean compile 'org.slf4j:slf4j-log4j12:1.7.25' I'll try to remove it.

Reply:
Yes, indeed! The comment was supposed to be added to the line after.